elastic/kibana
# [SecuritySolution] Query risk score without time range

## Summary
Issue:
https://github.com/elastic/security-team/issues/5817
Steps to verify:
1. visit app/security/entity_analytics
2. Inspect host risk score donut chart and user risk score donut chart
3. There should be no time range in their requests.
<img width="2543" alt="Screenshot 2023-01-27 at 12 21 01" src="https://user-images.githubusercontent.com/6295984/215087615-0b54a86e-b65f-4140-bebd-78f251693ba7.png">


### Checklist

Delete any items that are not applicable to this PR.
